app/controllers/candidate_interface/start_page_controller.rb

```diff
@@ -6,6 +6,24 @@ class StartPageController < CandidateInterfaceController
 
     def show; end
 
+    def account_prompt
```

Comment:
I'm not sure how I feel about `account_prompt`. I have to think for a second about what this means. If this was named `create_account_or_sign_in`, `sign_up_or_in` or something along those lines, I would know exactly what page this action is for. That being said, I feel weird about having an `or` in a controller action... 🤦‍♀ 

Another thought: could this be in its own controller? We have `SignInController` and `SignUpController`, what do you think about that idea?

Tbh, I think this is a personal preference thing, feel free to ignore this comment and leave as is.

Reply:
I can +1 either of those names, don't think the `or` is jarring. But I may be unfamiliar with Rails conventions that might say it's wrong? 🤔

Reply:
Agreed, naming can be better here. 

I think `create_account_or_sign_in` is the way to go. It's consistent with how we're referring to the feature in the Trello card and in the page title. 👍 

I like the separate controller idea - ironically, that's where I started when spiking out how this would work. I ended up moving it back into here because it seemed odd to set it apart from the start page and eligibility, given how tightly related they are in the UI. If anything though, eligibility should be in its own controller as well. I think I'll keep the actions here for now and extract into their own controller in a later refactor.

Reply:
🚢